plugins/module_utils/cmd_runner.py

```diff
@@ -129,8 +129,15 @@ def as_opt_eq_val(arg, ignore_none=None):
         return _ArgFormat(lambda value: ["{0}={1}".format(arg, value)], ignore_none=ignore_none)
 
     @staticmethod
-    def as_list(ignore_none=None):
-        return _ArgFormat(_ensure_list, ignore_none=ignore_none)
+    def as_list(ignore_none=None, min_len=1, max_len=None):
```

Comment:
What's the reason to make the minimum length 1 instead of 0?

Or will this never be called with a list of length 0?

Reply:
Temporary insanity. :rofl: 
Yeah, it definitely should be zero, because that list can come up empty from user parameters and what not.